samples/DotNet/Azure.Messaging.EventHubs/ManagedIdentityWebApp/SendReceive.aspx.cs

```diff
@@ -40,7 +40,7 @@ await using (EventHubProducerClient producerClient = new EventHubProducerClient(
         }
         protected async void btnReceive_Click(object sender, EventArgs e)
         {
-            await using (var consumerClient = new EventHubConsumerClient(EventHubConsumerClient.DefaultConsumerGroupName, $"{txtNamespace.Text}.servicebus.windows.net", txtEventHub.Text, new DefaultAzureCredential()))
+            await using (var consumerClient = new EventHubConsumerClient(EventHubConsumerClient.DefaultConsumerGroupName, txtNamespace.Text, txtEventHub.Text, new DefaultAzureCredential()))
```

Comment:
>txtNamespace.Text, [](start = 122, length = 18)

API doc says this needs to be an FQDN. Does user enter FQDN into namespace textbox on the page?

Reply:
@serkantkaraca - Yes. User enters the FQDN into the namespace text box now. I had updated the send code and tested it, but forgot to test the receive side. Thanks.